docs/token-api.md

```diff
@@ -34,69 +35,84 @@ parties, called `redeemers`, can invoke this operation.
-
-- `Wallet Manager`: A Wallet consists of a long-term identity and all its derivation (if any).
-  Examples of long-term identities are:
+## Token Management Service
+
+The `Token Management Service` (`token.ManagementService`) (TMS, for short) is the entry point of the Token SDK
+and gives access to the rest of the APIs.
+The tuple `(network, channel, namespace, public parameters)` uniquely identifies a TMS, where:
+- `network` is the identifier of the network/backend of reference;
+- `channel` is the channel inside the network. If not available, it is empty;
+- `namespace` is the namespace, inside the channel, that stores the tokens.
+- `public parameters` contain all information needed to operate the specific token infrastructure.
+
+The developer can get an instance of the `Token Management Service` by using the `GetManagementService` function:
+```go
+    tms := token.GetManagementService(context)
+```
+where `context` is an `FSC View Context`.
+The developer can pass additional options to request a specific TMS like:
+```go
+    tms := token.GetManagementService(context, token.WithNetwork("my-network"))
+```
+
+## Public Parameters Manager
+
+Each TMS is associated to some public parameters that contain all information needed to operate the token infrastructure.
+The `Public Parameters Manager` (`token.PublicParametersManager`) is responsible for handling the public parameters.
+Even though, parts of the public parameters are driver-specific, we can identify the following common information:
+    - `Precision`: The precision used to represent the token quantity.
+    - `MaxTokenValue`: It is the maximum quantity that a token can contain.
+    - `TokenDataHiding`: When true it means that the content of the tokens is hidden.
+    - `GraphHiding`:  When true it means that the tokens are untraceable.
+    - `Auditors`: A list of auditor identities.
+
+The developer can access the `Public Parameters Manager` of a give TMS as follows:
+```go
+    ppm := tms.PublicParametersManager()
+```
+
+## Wallet Manager
+
+A Wallet consists of a long-term identity and all its derivation (if any).
+Examples of long-term identities are:
     - An `X509 Certificate` for an ECDSA signing public-key
     - An `Idemix Credential`. In this case, the wallet will contain also all pseudonyms derived from the credential.
   
-  However, it is always the specific driver that dictates what a long-term identity is.
+However, it is always the specific driver that dictates what a long-term identity is.
 
-  All operations that require a signature refer to wallets to identify the signing and verification keys.
-  There are wallets for `Token Issuers`, `Owners`, and `Auditors`.
-  Depending on the nature of the wallet additional information can be extracted like:
+All operations that require a signature refer to wallets to identify the signing and verification keys.
+There are wallets for `Issuers`, `Owners`, and `Auditors`.
```

Comment:
Above we also introduced the roles of Certifiers. Why dont they have a specific wallet?